metrics_sdk/test/opentelemetry/sdk/metrics/meter_test.rb

```diff
@@ -185,8 +185,8 @@
       _(-> { meter.create_counter('1_counter') }).must_raise(INSTRUMENT_NAME_ERROR)
     end
 
-    it 'instrument name must not exceed 63 character limit' do
-      long_name = 'a' * 63
+    it 'instrument name must not exceed 256 character limit' do
+      long_name = 'a' * 256
       meter.create_counter(long_name)
       _(-> { meter.create_counter(long_name + 'a') }).must_raise(INSTRUMENT_NAME_ERROR)
     end
```

Comment:
If we decide to add the forward slash to the regex as part of this PR, the test on 194 should also be updated to include a forward slash.

Reply:
Looks great!